### ISSUE TITLE
GriefPrevention Bug with Region on FLAT World

### ISSUE THREAD

**Issue by d4rkmidian** [GriefPrevention/GriefPrevention#2153]
### Observed Behavior

When a region is defined players can't break or place blocks thus players cannot use the goldenshovel to create a claim area.
In FLAT World i tested, claims can be created inside the region but region is protected so players cannot break or place blocks.
It should't let players claim inside the region of a FLAT World. 
regions created /rg define <region-name>
all is default nothing changed, the server is fresh for test, to replicate things.
This happend to my main when i decide to have a creative FLAT, after i try to set a region (as usual) somehow inside the region claims can be done eventhough they can't place or break blocks that are protected from the region.

### Expected Behavior

Should behave like Normal Map.
When a region is presented players cannot be able to claim an area as expected.

### Reproduction steps

Here a video demonstrate
https://www.youtube.com/watch?v=PRoPHwlqpuw
_sorry for the background music_

### Stack trace or error log

```shell
[16:36:24] [ServerMain/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[16:36:28] [ServerMain/INFO]: Loaded 7 recipes
[16:36:30] [Server thread/INFO]: Starting minecraft server version 1.20.1
[16:36:30] [Server thread/INFO]: Loading properties
[16:36:31] [Server thread/INFO]: This server is running Paper version git-Paper-172 (MC: 1.20.1) (Implementing API version 1.20.1-R0.1-SNAPSHOT) (Git: 72e87ab)
[16:36:31] [Server thread/INFO]: Server Ping Player Sample Count: 12
[16:36:31] [Server thread/INFO]: Using 4 threads for Netty based IO
[16:36:31] [Server thread/WARN]: [!] The timings profiler has been enabled but has been scheduled for removal from Paper in the future.
    We recommend installing the spark profiler as a replacement: https://spark.lucko.me/
    For more information please visit: https://github.com/PaperMC/Paper/issues/8948
[16:36:31] [Server thread/INFO]: [ChunkTaskScheduler] Chunk system is using 1 I/O threads, 1 worker threads, and gen parallelism of 1 threads
[16:36:32] [Server thread/INFO]: Default game type: SURVIVAL
[16:36:32] [Server thread/INFO]: Generating keypair
[16:36:33] [Server thread/INFO]: Starting Minecraft server on *:16162
[16:36:33] [Server thread/INFO]: Using epoll channel type
[16:36:33] [Server thread/INFO]: Paper: Using libdeflate (Linux x86_64) compression from Velocity.
[16:36:33] [Server thread/INFO]: Paper: Using OpenSSL 1.1.x (Linux x86_64) cipher from Velocity.
[16:36:36] [Server thread/INFO]: [LuckPerms] Loading server plugin LuckPerms v5.4.102
[16:36:37] [Server thread/INFO]: [Vault] Loading server plugin Vault v1.7.3-b131
[16:36:37] [Server thread/INFO]: [WorldEdit] Loading server plugin WorldEdit v7.2.15+6463-5ca4dff
[16:36:39] [Server thread/INFO]: Got request to register class com.sk89q.worldedit.bukkit.BukkitServerInterface with WorldEdit [com.sk89q.worldedit.extension.platform.PlatformManager@5fd498c6]
[16:36:39] [Server thread/INFO]: [Essentials] Loading server plugin Essentials v2.20.1
[16:36:39] [Server thread/INFO]: [WorldGuard] Loading server plugin WorldGuard v7.0.9+5934e49
[16:36:39] [Server thread/INFO]: [Multiverse-Core] Loading server plugin Multiverse-Core v4.3.11
[16:36:39] [Server thread/INFO]: [EssentialsSpawn] Loading server plugin EssentialsSpawn v2.20.1
[16:36:39] [Server thread/INFO]: [GriefPrevention] Loading server plugin GriefPrevention v16.18.1-7-g64b348a
[16:36:39] [Server thread/INFO]: [ProtocolLib] Loading server plugin ProtocolLib v5.0.0-SNAPSHOT-642
[16:36:39] [Server thread/WARN]: [ProtocolLib] Version (MC: 1.20.1) has not yet been tested! Proceed with caution.
[16:36:43] [Server thread/INFO]: Server permissions file permissions.yml is empty, ignoring it
[16:36:43] [Server thread/INFO]: [LuckPerms] Enabling LuckPerms v5.4.102
[16:36:44] [Server thread/INFO]:         __    
[16:36:44] [Server thread/INFO]:   |    |__)   LuckPerms v5.4.102
[16:36:44] [Server thread/INFO]:   |___ |      Running on Bukkit - Paper
[16:36:44] [Server thread/INFO]: 
[16:36:44] [Server thread/INFO]: [LuckPerms] Loading configuration...
[16:36:45] [Server thread/INFO]: [LuckPerms] Loading storage provider... [H2]
[16:36:46] [Server thread/INFO]: [LuckPerms] Loading internal permission managers...
[16:36:46] [Server thread/INFO]: [LuckPerms] Performing initial data load...
[16:36:47] [Server thread/INFO]: [LuckPerms] Successfully enabled. (took 4359ms)
[16:36:47] [Server thread/INFO]: [Vault] Enabling Vault v1.7.3-b131
[16:36:47] [Server thread/INFO]: [Vault] [Economy] Essentials Economy found: Waiting
[16:36:47] [Server thread/INFO]: [Vault] [Permission] SuperPermissions loaded as backup permission system.
[16:36:47] [Server thread/INFO]: [Vault] Enabled Version 1.7.3-b131
[16:36:47] [Server thread/INFO]: [LuckPerms] Registered Vault permission & chat hook.
[16:36:47] [Server thread/INFO]: [WorldEdit] Enabling WorldEdit v7.2.15+6463-5ca4dff
[16:36:47] [Server thread/INFO]: Registering commands with com.sk89q.worldedit.bukkit.BukkitServerInterface
[16:36:48] [Server thread/INFO]: WEPIF: Vault detected! Using Vault for permissions
[16:36:49] [Server thread/INFO]: Using com.sk89q.worldedit.bukkit.adapter.impl.v1_20_R1.PaperweightAdapter as the Bukkit adapter
[16:36:53] [Server thread/INFO]: [ProtocolLib] Enabling ProtocolLib v5.0.0-SNAPSHOT-642
[16:36:53] [Server thread/WARN]: **** SERVER IS RUNNING IN OFFLINE/INSECURE MODE!
[16:36:53] [Server thread/WARN]: The server will make no attempt to authenticate usernames. Beware.
[16:36:53] [Server thread/WARN]: While this makes the game possible to play without internet access, it also opens up the ability for hackers to connect with any username they choose.
[16:36:53] [Server thread/WARN]: To change this, set "online-mode" to "true" in the server.properties file.
[16:36:53] [Server thread/INFO]: Preparing level "survival"
[16:36:55] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[16:36:56] [Server thread/INFO]: Time elapsed: 406 ms
[16:36:56] [Server thread/INFO]: Preparing start region for dimension minecraft:the_nether
[16:36:56] [Server thread/INFO]: Time elapsed: 49 ms
[16:36:56] [Server thread/INFO]: Preparing start region for dimension minecraft:the_end
[16:36:56] [Server thread/INFO]: Time elapsed: 54 ms
[16:36:56] [Server thread/INFO]: [Essentials] Enabling Essentials v2.20.1
[16:36:57] [Server thread/INFO]: [Essentials] Attempting to convert old kits in config.yml to new kits.yml
[16:36:57] [Server thread/INFO]: [Essentials] No kits found to migrate.
[16:36:58] [Server thread/INFO]: [Essentials] Loaded 39094 items from items.json.
[16:36:58] [Server thread/INFO]: [Essentials] Using locale en_US
[16:36:59] [Server thread/INFO]: [Essentials] ServerListPingEvent: Spigot iterator API
[16:36:59] [Server thread/INFO]: [Essentials] Starting Metrics. Opt-out using the global bStats config.
[16:36:59] [Server thread/INFO]: [Vault] [Economy] Essentials Economy hooked.
[16:36:59] [Server thread/INFO]: [Essentials] Using Vault based permissions (LuckPerms)
[16:36:59] [Server thread/INFO]: [WorldGuard] Enabling WorldGuard v7.0.9+5934e49
[16:37:00] [Server thread/INFO]: [WorldGuard] (survival) TNT ignition is PERMITTED.
[16:37:00] [Server thread/INFO]: [WorldGuard] (survival) Lighters are PERMITTED.
[16:37:00] [Server thread/INFO]: [WorldGuard] (survival) Lava fire is PERMITTED.
[16:37:00] [Server thread/INFO]: [WorldGuard] (survival) Fire spread is UNRESTRICTED.
[16:37:00] [Server thread/INFO]: [WorldGuard] Loaded configuration for world 'survival'
[16:37:00] [Server thread/INFO]: [WorldGuard] (survival_nether) TNT ignition is PERMITTED.
[16:37:00] [Server thread/INFO]: [WorldGuard] (survival_nether) Lighters are PERMITTED.
[16:37:00] [Server thread/INFO]: [WorldGuard] (survival_nether) Lava fire is PERMITTED.
[16:37:00] [Server thread/INFO]: [WorldGuard] (survival_nether) Fire spread is UNRESTRICTED.
[16:37:00] [Server thread/INFO]: [WorldGuard] Loaded configuration for world 'survival_nether'
[16:37:00] [Server thread/INFO]: [WorldGuard] (survival_the_end) TNT ignition is PERMITTED.
[16:37:00] [Server thread/INFO]: [WorldGuard] (survival_the_end) Lighters are PERMITTED.
[16:37:00] [Server thread/INFO]: [WorldGuard] (survival_the_end) Lava fire is PERMITTED.
[16:37:00] [Server thread/INFO]: [WorldGuard] (survival_the_end) Fire spread is UNRESTRICTED.
[16:37:00] [Server thread/INFO]: [WorldGuard] Loaded configuration for world 'survival_the_end'
[16:37:00] [Server thread/INFO]: [WorldGuard] Loading region data...
[16:37:01] [Server thread/INFO]: [Multiverse-Core] Enabling Multiverse-Core v4.3.11
[16:37:01] [Server thread/WARN]: [Multiverse-Core] "Multiverse-Core v4.3.11" has registered a listener for org.bukkit.event.entity.EntityCreatePortalEvent on method "public void com.onarandombox.MultiverseCore.listeners.MVPortalListener.entityPortalCreate(org.bukkit.event.entity.EntityCreatePortalEvent)", but the event is Deprecated. "Server performance will be affected"; please notify the authors [dumptruckman, Rigby, fernferret, lithium3141, main--].
[16:37:01] [Server thread/INFO]: [Multiverse-Core] §aWe are aware of the warning about the deprecated event. There is no alternative that allows us to do what we need to do and performance impact is negligible. It is safe to ignore.
[16:37:02] [Server thread/INFO]: Preparing start region for dimension minecraft:flat
[16:37:04] [Server thread/INFO]: Time elapsed: 1757 ms
[16:37:04] [Server thread/INFO]: [WorldGuard] (flat) TNT ignition is PERMITTED.
[16:37:04] [Server thread/INFO]: [WorldGuard] (flat) Lighters are PERMITTED.
[16:37:04] [Server thread/INFO]: [WorldGuard] (flat) Lava fire is PERMITTED.
[16:37:04] [Server thread/INFO]: [WorldGuard] (flat) Fire spread is UNRESTRICTED.
[16:37:04] [Server thread/INFO]: [WorldGuard] Loaded configuration for world 'flat'
[16:37:05] [Server thread/INFO]: Preparing start region for dimension minecraft:creative
[16:37:05] [Server thread/INFO]: Time elapsed: 565 ms
[16:37:05] [Server thread/INFO]: [WorldGuard] (creative) TNT ignition is PERMITTED.
[16:37:05] [Server thread/INFO]: [WorldGuard] (creative) Lighters are PERMITTED.
[16:37:05] [Server thread/INFO]: [WorldGuard] (creative) Lava fire is PERMITTED.
[16:37:05] [Server thread/INFO]: [WorldGuard] (creative) Fire spread is UNRESTRICTED.
[16:37:05] [Server thread/INFO]: [WorldGuard] Loaded configuration for world 'creative'
[16:37:05] [Server thread/INFO]: [Multiverse-Core] 5 - World(s) loaded.
[16:37:05] [Server thread/WARN]: [Multiverse-Core] Buscript failed to load! The script command will be disabled! If you would like not to see this message, use `/mv conf enablebuscript false` to disable Buscript from loading.
[16:37:05] [Server thread/INFO]: [Multiverse-Core] Version 4.3.11 (API v24) Enabled - By dumptruckman, Rigby, fernferret, lithium3141 and main--
[16:37:05] [Server thread/INFO]: [EssentialsSpawn] Enabling EssentialsSpawn v2.20.1
[16:37:05] [Server thread/INFO]: [EssentialsSpawn] Starting Metrics. Opt-out using the global bStats config.
[16:37:05] [Server thread/INFO]: [GriefPrevention] Enabling GriefPrevention v16.18.1-7-g64b348a
[16:37:06] [Server thread/INFO]: [GriefPrevention] Finished loading configuration.
[16:37:06] [Server thread/INFO]: [GriefPrevention] 1 total claims loaded.
[16:37:07] [Server thread/INFO]: [GriefPrevention] Customizable messages loaded.
[16:37:07] [Server thread/INFO]: [GriefPrevention] Successfully hooked into WorldGuard.
[16:37:07] [Server thread/INFO]: [GriefPrevention] Finished loading data (File Mode).
[16:37:07] [Server thread/INFO]: [GriefPrevention] Boot finished.
[16:37:08] [Server thread/INFO]: Starting GS4 status listener
[16:37:08] [Server thread/INFO]: Thread Query Listener started
[16:37:08] [Server thread/INFO]: JMX monitoring enabled
[16:37:08] [Query Listener #1/INFO]: Query running on 0.0.0.0:9898
[16:37:08] [Server thread/INFO]: Running delayed init tasks
[16:37:08] [Server thread/INFO]: [Essentials] Essentials found a compatible payment resolution method: Vault Compatibility Layer (v1.7.3-b131)!
[16:37:08] [Server thread/INFO]: Done (38.062s)! For help, type "help"
[16:37:08] [Server thread/INFO]: Timings Reset
[16:37:08] [Craft Scheduler Thread - 4 - Essentials/INFO]: [Essentials] Fetching version information...
[16:37:08] [Craft Scheduler Thread - 6 - Vault/INFO]: [Vault] Checking for Updates ... 
[16:37:09] [Craft Scheduler Thread - 6 - Vault/INFO]: [Vault] No new version available
[16:37:18] [Server thread/INFO]: Server Plugins (9):
[16:37:18] [Server thread/INFO]: Bukkit Plugins:
[16:37:18] [Server thread/INFO]:  - Essentials, EssentialsSpawn, GriefPrevention, LuckPerms, Multiverse-Core, ProtocolLib, Vault, WorldEdit, WorldGuard
[16:38:00] [User Authenticator #0/INFO]: UUID of player d4rkmidian is 81baa555-7ebb-3c82-bcff-00a217bb5991
[16:38:00] [Server thread/INFO]: d4rkmidian joined the game
[16:38:01] [Server thread/INFO]: d4rkmidian[/**.**.**.**:59616] logged in with entity id 228 at ([survival]-45.43022693923296, 70.0, -151.7622765409656)
[16:38:01] [Server thread/WARN]: [Essentials] Permissions lag notice with (LuckPermsHandler). Response took 64.668333ms. Summary: Getting prefix for d4rkmidian
[16:38:09] [Server thread/INFO]: d4rkmidian issued server command: /gm 1
[16:38:16] [Server thread/INFO]: d4rkmidian issued server command: /mvtp flat
[16:38:28] [Server thread/INFO]: d4rkmidian issued server command: /ggm 1
[16:38:29] [Server thread/INFO]: d4rkmidian issued server command: /gm 1
```

### Server version

```shell
This server is running Paper version git-Paper-172 (MC: 1.20.1) (Implementing API version 1.20.1-R0.1-SNAPSHOT) (Git: 72e87ab)
```

### GriefPrevention version

```shell
GriefPrevention version 16.18.1-7-g64b348a
```

### Configuration

```yaml
# Default values are perfect for most servers.  If you want to customize and have a question, look for the answer here first: http://dev.bukkit.org/bukkit-plugins/grief-prevention/pages/setup-and-configuration/

GriefPrevention:
  SeaLevelOverrides:
    survival: -1
    survival_nether: -1
    survival_the_end: -1
    flat: -1
    creative: -1
  Claims:
    Mode:
      survival_nether: Survival
      survival_the_end: Survival
      creative: Survival
      survival: Survival
      flat: Survival
    PreventGlobalMonsterEggs: true
    PreventTheft: true
    ProtectCreatures: true
    PreventButtonsSwitches: true
    LockWoodenDoors: false
    LockTrapDoors: false
    LockFenceGates: true
    EnderPearlsRequireAccessTrust: true
    RaidTriggersRequireBuildTrust: true
    ProtectHorses: true
    ProtectDonkeys: true
    ProtectLlamas: true
    InitialBlocks: 100
    Claim Blocks Accrued Per Hour:
      Default: 100
    Max Accrued Claim Blocks:
      Default: 80000
    Accrued Idle Threshold: 0
    AccruedIdlePercent: 0
    AbandonReturnRatio: 1.0
    AutomaticNewPlayerClaimsRadius: -1
    AutomaticNewPlayerClaimsRadiusMinimum: 0
    ExtendIntoGroundDistance: 5
    MinimumWidth: 5
    MinimumArea: 100
    MaximumDepth: -2147483648
    InvestigationTool: STICK
    ModificationTool: GOLDEN_SHOVEL
    Expiration:
      ChestClaimDays: 7
      UnusedClaimDays: 14
      AllClaims:
        DaysInactive: 60
        ExceptWhenOwnerHasTotalClaimBlocks: 10000
        ExceptWhenOwnerHasBonusClaimBlocks: 5000
      AutomaticNatureRestoration:
        SurvivalWorlds: false
    AllowTrappedInAdminClaims: false
    MaximumNumberOfClaimsPerPlayer: 0
    CreationRequiresWorldGuardBuildPermission: true
    VillagerTradingRequiresPermission: true
    CommandsRequiringAccessTrust: /sethome
    DeliverManuals: true
    ManualDeliveryDelaySeconds: 30
    RavagersBreakBlocks: true
    FireSpreadsInClaims: false
    FireDamagesInClaims: false
    LecternReadingRequiresAccessTrust: true
  Spam:
    Enabled: true
    LoginCooldownSeconds: 60
    LoginLogoutNotificationsPerMinute: 5
    ChatSlashCommands: /me;/global;/local
    WhisperSlashCommands: /tell;/pm;/r;/whisper;/msg
    WarningMessage: Please reduce your noise level.  Spammers will be banned.
    BanOffenders: true
    BanMessage: Banned for spam.
    AllowedIpAddresses: 1.2.3.4; 5.6.7.8
    DeathMessageCooldownSeconds: 120
    Logout Message Delay In Seconds: 0
  PvP:
    RulesEnabledInWorld:
      survival: true
      survival_nether: true
      survival_the_end: true
      flat: true
      creative: true
    ProtectFreshSpawns: true
    PunishLogout: true
    CombatTimeoutSeconds: 15
    AllowCombatItemDrop: false
    BlockedSlashCommands: /home;/vanish;/spawn;/tpa
    ProtectPlayersInLandClaims:
      PlayerOwnedClaims: true
      AdministrativeClaims: true
      AdministrativeSubdivisions: true
    AllowLavaDumpingNearOtherPlayers:
      PvPWorlds: true
      NonPvPWorlds: false
    AllowFlintAndSteelNearOtherPlayers:
      PvPWorlds: true
      NonPvPWorlds: false
    ProtectPetsOutsideLandClaims: false
  Economy:
    ClaimBlocksMaxBonus: 0
    ClaimBlocksPurchaseCost: 0.0
    ClaimBlocksSellValue: 0.0
  ProtectItemsDroppedOnDeath:
    PvPWorlds: false
    NonPvPWorlds: true
  BlockLandClaimExplosions: true
  BlockSurfaceCreeperExplosions: true
  BlockSurfaceOtherExplosions: true
  LimitSkyTrees: true
  LimitTreeGrowth: false
  PistonMovement: CLAIMS_ONLY
  PistonExplosionSound: true
  FireSpreads: false
  FireDestroys: false
  AdminsGetWhispers: true
  AdminsGetSignNotifications: true
  VisualizationAntiCheatCompatMode: false
  SmartBan: true
  Mute New Players Using Banned Words: true
  MaxPlayersPerIpAddress: 3
  SilenceBans: true
  Siege:
    Worlds: []
    BreakableBlocks:
    - GRASS_BLOCK
    - DIRT
    - COBBLESTONE
    - OAK_PLANKS
    - SPRUCE_PLANKS
    - BIRCH_PLANKS
    - JUNGLE_PLANKS
    - ACACIA_PLANKS
    - DARK_OAK_PLANKS
    - SAND
    - GRAVEL
    - GLASS
    - GRASS
    - FERN
    - DEAD_BUSH
    - WHITE_WOOL
    - ORANGE_WOOL
    - MAGENTA_WOOL
    - LIGHT_BLUE_WOOL
    - YELLOW_WOOL
    - LIME_WOOL
    - PINK_WOOL
    - GRAY_WOOL
    - LIGHT_GRAY_WOOL
    - CYAN_WOOL
    - PURPLE_WOOL
    - BLUE_WOOL
    - BROWN_WOOL
    - GREEN_WOOL
    - RED_WOOL
    - BLACK_WOOL
    - SNOW
    - GLASS_PANE
    DoorsOpenDelayInSeconds: 300
    CooldownEndInMinutes: 60
  EndermenMoveBlocks: false
  SilverfishBreakBlocks: false
  CreaturesTrampleCrops: false
  RabbitsEatCrops: true
  HardModeZombiesBreakDoors: false
  Database:
    URL: ''
    UserName: ''
    Password: ''
  UseBanCommand: false
  BanCommandPattern: ban %name% %reason%
  Advanced:
    fixNegativeClaimblockAmounts: true
    ClaimExpirationCheckRate: 60
    OfflinePlayer_cache_days: 90
  Abridged Logs:
    Days To Keep: 7
    Included Entry Types:
      Social Activity: true
      Suspicious Activity: true
      Administrative Activity: false
      Debug: false
      Muted Chat Messages: false
  ConfigVersion: 1
```

### Plugin list

```shell
Essentials, EssentialsSpawn, GriefPrevention, LuckPerms, Multiverse-Core, ProtocolLib, Vault, WorldEdit, WorldGuard
```

### Running without GriefPrevention

- [ ] I attempted running the server without GriefPrevention installed.
- [ ] The problem does _not_ occur when GriefPrevention is removed from the server.

### Running with only GriefPrevention

- [ ] I attempted running only GriefPrevention on the server.
- [ ] The issue still occurs when GriefPrevention is the only plugin running.

### Running on a fresh, clean server installation

- [X] I attempted testing for the issue on a new server.
- [X] The issue still occurs on a new server.

### Using unmodified client

- [x] I attempted testing for the issue with the vanilla client.
- [x] The issue still occurs when using the vanilla client.

### We appreciate you taking the time to fill out a bug report!

- [X] I searched for similar issues before submitting this bug report.

**Comment by RoboMWM** [comment]
Use an admin claim to prevent players from claiming land.

Worldguard integration is funky, you have to disallow passthrough flag. I also plan to remove dependency on WorldGuard in GP in the next major version, so I'd advise not relying on WorldGuard to prevent players from creating claims.

**Comment by d4rkmidian** [comment]
i dont know why on FLAT world in particular this happening...
the regions doesnt have any flags written in on the test.

i use region flags in general in my smp for example in one spawn.
use: allow
item-drop: deny
fall-damage: deny
item-pickup: deny
pickup-drops: deny
potion-splash: deny
mob-spawning: deny

so wg is kinda handy.

PS.
i would like to keep
  CreateClaimFailOverlapRegion:
    Text: You can't claim all of this because you're not allowed to build here.

**Comment by RoboMWM** [comment]
Idk either, maybe has something to do with height limit. Either way, if you're creating worldguard regions, I'd imagine it's not that much more to also create an adminclaim? If it is, I wonder if an addon can easily simultaneously create adminclaims when worldguard regions are defined.

**Comment by Jikoo** [comment]
WG support has been dropped from v17.

**Comment by d4rkmidian** [comment]
How is this unverified?? I tested on a fresh server back then and made a video about it.
Define a region on a flat world with multiverse, players are able to build there.
And the flat world is creative, 1 map nor nether or end. 
Another single map is also a normal creative, i've set a region and working there,  players can't build in the region of normal single map creative. Survival maps that have also nether and end (I don't know if that's matter) is also working. Only on flat world. And I tested twice.
